en/airframes/architecture.md

```diff
@@ -1,28 +0,0 @@
-# Architectural Overview
-
-PX4 is architected in a modular fashion, such that there is a single codebase for all vehicle types.
-
-{% mermaid %}
```

Comment:
@bkueng FYI, I deleted this doc because it is pointless
-  The "architecture" says very little 
- the basic equipment duplicates info here: https://dev.px4.io/en/setup/config_initial.html#basic-equipment

The only thing of use is this mermaid doc, which I will discuss elsewhere.